recipes-core/rcu-sanity-check/files/post-update-sanity-check

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+rm /data/rcu-service/firmware/update-staged
+rm /etc/rcu-mender-image-firstboot
+
+RCU_SERVICE_STATUS=$(systemctl is-active rcu-service)
+ETH0_STATUS=$(cat /sys/class/net/eth0/operstate)
```

Comment:
This is here in case we want to also sanity check eth0 is up.

Reply:
If eth0 is not connected for any reason (e.g. customer use eth1 instead). Will this detect a false failure?

Reply:
@kengsooncheah I think it will- this is part of the reason I decided to not include the check for now. Probably more work needed to get this right.